benefits/eligibility/views.py

```diff
@@ -122,6 +128,7 @@ def start(request):
     ctx["start_headline"] = _(verifier.start_headline)
     ctx["start_sub_headline"] = _(verifier.start_sub_headline)
     ctx["media"] = media
+    ctx["info_link"] = f"{reverse(ROUTE_HELP)}#what-is-cal-itp"
```

Comment:
```suggestion
```

This was removed in [#956](https://github.com/cal-itp/benefits/pull/956/files#diff-bd503c63c9a7b07070e81c9df074adafc509da84db6e270d3c91c54e282218b2L131) (merge conflict probably put it back).